src/components/icons/ExternalLink.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
+export let size = 24;
+</script>
+
+<style>
+path {
```

Comment:
I tend to just add a `fill=currentColor` in the svg itself instead of using the style tag here so we don't introduce more classes. I'm not sure it matters in the end, and perhaps the best approach is a class on the svg that always sets `fill=currentColor`